programs/marginfi/src/state/native_oracle/mod.rs

```diff
@@ -0,0 +1,18 @@
+use self::pyth_crosschain::PythnetPriceFeed;
+
+pub mod pyth_crosschain;
+
+#[repr(C, align(8))]
```

Comment:
if using `align(8)` might as well make this repr(u64) as well to ensure the first 8 bytes are properly initialized